lisa/tools/fips.py

```diff
@@ -0,0 +1,379 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from abc import ABC, abstractmethod
+from pathlib import PurePosixPath
+from typing import TYPE_CHECKING, Any
+
+from assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import CBLMariner, OperatingSystem
+from lisa.tools import Blkid, Cat, Sed
+from lisa.util import UnsupportedDistroException, get_matched_str
+
+if TYPE_CHECKING:
+    from lisa.operating_system import Posix
+    from lisa.node import Node
+
+
+def assert_package_existance(
```

Comment:
you can extend the `os.package_exists` method to support it for all places. It may be like `def package_exists(self, package: Union[str, Tool, Type[Tool]], raise_error: bool = False) -> bool: